Marlin/UBL_G29.cpp

```diff
@@ -712,58 +712,193 @@
           z_values[x][y] += constant;
   }
 
+  bool position_is_reachable_UBL(float X_target, float Y_target , bool is_probe) {

```

Comment:
I would rather we use the `position_is_reachable` function, and not create a whole separate one for this one janky feature.